pkg/interop/storage/storage.go

```diff
@@ -28,6 +28,10 @@ const (
 	ValuesOnly FindFlags = 1 << 2
 	// DeserializeValues is used for values deserializing.
 	DeserializeValues FindFlags = 1 << 3
+	// PickField0 is used to get first field in a serialized struct.
```

Comment:
...or array? I think both will work the same.

Reply:
Fixed.